python/features/steps/general_purpose_steps.py

```diff
@@ -8,20 +8,6 @@
 use_step_matcher("cfparse")
 
 
-def get_model_file(context, path: str) -> str:
```

Comment:
Good clean up find, if I am understanding correctly the logic here and execution in GitPod, the update to the path for the files in the feature files let this get dropped.

Reply:
Ooooh looks like this also helped allow for individual feature file runs